user-manuals/data-manager/configurations.md

```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
 Config ID cannot be changed after creation. Create a new Config if a new Config ID is needed.
 {% endhint %}
 
+## Verify the Data flow of Data Configurations
+
+In HotWax Commerce, the significance of the Data Manager Configurations feature extends beyond the management of data flow. A crucial aspect of this tool lies in the Data Manager Logs, which play a pivotal role in ensuring the accuracy and completeness of data imported into the Order Management System (OMS). These logs provide essential insights into the status of imported data, allowing users to verify the success of their operations and identify potential issues promptly.
+
+### Steps
+
+
+1. Search for the Desired Service:
```

Comment:
Search is keyword based we should not bind it with service, user may not know the service name sometimes.